common/api/ui-framework.api.md

```diff
@@ -1910,7 +1910,7 @@ export const expandWidget: (base: {
         readonly home: {
             readonly widgetIndex: number;
             readonly widgetId: string | undefined;
-            readonly side: PanelSide;
+            readonly side: "top" | "left" | "bottom" | "right";

```

Comment:
In ui-framework, run "npm run clean", "npm run build" then "npm run extract-api". That will cut down on the false differences.